tests/validation/tests/v3_api/test_rke_cluster_provisioning.py

```diff
@@ -1069,3 +1130,21 @@ def create_custom_host_from_nodes(nodes, node_roles,
                                      check_intermediate_state=False)
 
     return cluster, nodes
+
+
+def get_cis_rke_config(profile=CIS_SCAN_PROFILE):
```

Comment:
Can we have this in common.py? As this is used by 2 files

Reply:
Do we really want to move all the `rke_config_cis_*` configuration logic to `common.py` too?

Reply:
Ahh this is  getting `rke_config`